libs/ai-endpoints/langchain_nvidia_ai_endpoints/reranking.py

```diff
@@ -35,6 +35,13 @@ class Config:
     )
     top_n: int = Field(5, ge=0, description="The number of documents to return.")
     model: Optional[str] = Field(description="The model to use for reranking.")
+    truncate: Optional[Literal["NONE", "END"]] = Field(
+        description=(
```

Comment:
Is it appropriate to have a default value of `None` to reflect the schema?
https://nim-docs-staging.s3.us-west-1.amazonaws.com/text-reranking/main/_static/html/ranking.html#tag/ranking/operation/rank

Reply:
(i can't access that link)

the `nv-rerank-qa-mistral-4b:1` endpoint doesn't support `truncate`. letting the param be optional lets the server side default come through.

the alternative is to make `truncate="NONE"` as default, add a special case in the code for endpoints that do not support the param and remove it w/ a warning to the user.

https://docs.api.nvidia.com/nim/reference/nvidia-reranking-4b-infer
![image](https://github.com/user-attachments/assets/3ddb6a01-6e08-438c-92ca-78ca6ab0b830)